src/test/scala/wdl4s/expression/FileEvaluatorSpec.scala

```diff
@@ -45,12 +42,13 @@ class FileEvaluatorSpec extends FlatSpec with Matchers {
     (""" [read_string("x"), read_string("y")] """, Seq(WdlSingleFile("x"), WdlSingleFile("y")), WdlArrayType(WdlStringType)),
     (""" {read_int("a"): read_string("x"), 4: read_string("y")} """, Seq(WdlSingleFile("a"), WdlSingleFile("x"), WdlSingleFile("y")), WdlArrayType(WdlStringType)),
     (""" glob("out-*.txt") """, Seq(WdlGlobFile("out-*.txt")), WdlFileType),
+    (""" glob("out-*.txt")[0] """, Seq(WdlGlobFile("out-*.txt")), WdlFileType),
```

Comment:
I don't know of any that would actually pass parsing or generate a proper AST. Filing a new ticket.